build.ps1

```diff
@@ -234,12 +234,7 @@ if (Test-Path $ConfigFile) {
     }
 }
 
-$DotNetHome = if ($env:DOTNET_HOME) { $env:DOTNET_HOME } `
-    elseif ($CI) { Join-Path $PSScriptRoot '.dotnet' } `
-    elseif ($env:USERPROFILE) { Join-Path $env:USERPROFILE '.dotnet'} `
-    elseif ($env:HOME) {Join-Path $env:HOME '.dotnet'}`
-    else { Join-Path $PSScriptRoot '.dotnet'}
-
+$DotNetHome = Join-Path $PSScriptRoot '.dotnet'
 $env:DOTNET_HOME = $DotNetHome
```

Comment:
Why do we still need both `$env:DOTNET_ROOT` and `$env:DOTNET_HOME`?  I realize they have different values on Windows but don't `dotnet` and our build scripts do the **Right Thing:tm:** with one of these?

Put another way, could we be consistent and use only one of these?

Reply:
DOTNET_HOME is a thing our team invented years ago (like 1.0 days), but the SDK team decided not to adopt. We'll get rid of it in #7280. KoreBuild uses this env var to pick the location of where SDKs and build tools are installed.

There is a difference on Windows. Because we support x64 and x86 SDKs, DOTNET_HOME=$repoRoot/.dotnet, DOTNET_ROOT=$DOTNET_HOME/$ARCH